tests/test_code_gen.py

```diff
@@ -476,6 +476,15 @@ def test_non_string_leakage(self):
         '''
         self.assertAstRoundtrips(source)
 
+    def test_issue_89(self):
```

Comment:
Could you please give this a bit more descriptive name? I was lazy when I tried to reproduce the problem so I went with ` test_issue_89` :)